lib/Runtime/Language/JavascriptExceptionOperators.cpp

```diff
@@ -240,20 +251,22 @@ namespace Js
         JavascriptExceptionObject *exception = nullptr;
 
         PROBE_STACK(scriptContext, Constants::MinStackDefault + argsSize);
-        Js::JavascriptExceptionOperators::TryCatchFrameAddrStack tryCatchFrameAddrStack(scriptContext, framePtr);
-
-        try
         {
-            Js::JavascriptExceptionOperators::AutoCatchHandlerExists autoCatchHandlerExists(scriptContext);
+            Js::JavascriptExceptionOperators::TryCatchFrameAddrStack tryCatchFrameAddrStack(scriptContext, framePtr);
+
+            try
+            {
+                Js::JavascriptExceptionOperators::AutoCatchHandlerExists autoCatchHandlerExists(scriptContext);
 #if defined(_M_ARM)
-            continuation = arm_CallEhFrame(tryAddr, framePtr, localsPtr, argsSize);
+                continuation = arm_CallEhFrame(tryAddr, framePtr, localsPtr, argsSize);
 #elif defined(_M_ARM64)
-            continuation = arm64_CallEhFrame(tryAddr, framePtr, localsPtr, argsSize);
+                continuation = arm64_CallEhFrame(tryAddr, framePtr, localsPtr, argsSize);
 #endif
-        }
-        catch (const Js::JavascriptException& err)
-        {
-            exception = err.GetAndClear();
+            }
+            catch (const Js::JavascriptException& err)
+            {
+                exception = err.GetAndClear();
```

Comment:
>exception = err.GetAndClear(); [](start = 16, length = 30)

nit: For x86/x64 you get the tryCatchFrameAddr in catch block but not here. Please be consistent.

Reply:
Done.